locales/fr_FR/LC_MESSAGES/messages.po

```diff
@@ -466,7 +466,7 @@ msgid "Language"
 msgstr "Langue"
 
 msgid "Desktop version"
-msgstr ""
+msgstr "Version bureau"
```

Comment:
I think, that following translation would be nicer:
- `Désactiver la version mobile`
- `Site web`
- `Site pour PC`

Reply:
Yes these sound better, "Site pour PC" would be my preferred one.

Reply:
@ferrocarta so please add that change to your PR, preferably squashed into the existing commit.

Reply:
Hopefully I did it right, this is my first attempt at using Github, please forgive me if I make mistakes.